--- origsrc/lesstif-0.95.0/ac_debug.m4	2004-02-01 09:49:40.000000000 -0600
+++ src/lesstif-0.95.0/ac_debug.m4	2008-10-26 22:18:45.100015900 -0500
@@ -4,7 +4,7 @@
 dnl Source code which depends on this is mostly in
 dnl DebugUtil.c/.h
 dnl
-AC_DEFUN([LT_WITH_DMALLOC],
+AC_DEFUN([LESSTIF_WITH_DMALLOC],
 [AC_MSG_CHECKING(if malloc debugging is wanted)
 AC_ARG_WITH(dmalloc,
 [  --with-dmalloc[=path]   use dmalloc, see INSTALL(.html) for reference],
@@ -47,7 +47,7 @@
 dnl Source code which depends on this is mostly in
 dnl DebugUtil.c/.h
 dnl
-AC_DEFUN([LT_WITH_DBMALLOC],
+AC_DEFUN([LESSTIF_WITH_DBMALLOC],
 [AC_MSG_CHECKING(if malloc debugging is wanted)
 AC_ARG_WITH(dbmalloc,
 [  --with-dbmalloc[=path]   use dbmalloc, see INSTALL(.html) for reference],
--- origsrc/lesstif-0.95.0/ac_have_libxp.m4	2004-02-01 09:49:40.000000000 -0600
+++ src/lesstif-0.95.0/ac_have_libxp.m4	2008-10-26 21:55:45.928140900 -0500
@@ -11,7 +11,7 @@
 dnl Makefiles. Perhaps one should immediately add those libs
 dnl to link commands which include libXm version2.1?!
 dnl
-AC_DEFUN([LT_HAVE_LIBXP],
+AC_DEFUN([LESSTIF_HAVE_LIBXP],
 [AC_REQUIRE([AC_PATH_X])
 AC_CACHE_CHECK(whether libXp is available, lt_cv_libxp,
 [lt_save_CFLAGS="$CFLAGS"
@@ -34,9 +34,9 @@
 ])
 if test $lt_cv_libxp = yes; then
   AC_DEFINE(HAVE_LIB_XP)
-  LT_HAVELIBXP=1
+  LESSTIF_HAVELIBXP=1
 else
-  LT_HAVELIBXP=0 
+  LESSTIF_HAVELIBXP=0 
 fi
 AM_CONDITIONAL(Have_Libxp, test "$lt_cv_libxp" = "yes")
 AC_SUBST(LT_HAVELIBXP)
--- origsrc/lesstif-0.95.0/acinclude.m4	2005-11-01 02:14:56.000000000 -0600
+++ src/lesstif-0.95.0/acinclude.m4	2008-10-27 21:50:37.638420900 -0500
@@ -330,7 +330,7 @@
 dnl
 dnl Extended check for gethostname()
 dnl
-AC_DEFUN([LT_HAVE_GETHOSTNAME],
+AC_DEFUN([LESSTIF_HAVE_GETHOSTNAME],
 [AC_REQUIRE([AC_PATH_X])
 AC_CACHE_CHECK(whether gethostname() is available, lt_cv_gethostname,
 [lt_save_CFLAGS="$CFLAGS"
@@ -369,7 +369,7 @@
 dnl We use our own version of Debian's man2html.
 dnl
 
-AC_DEFUN([LT_HAVE_MAN2HTML],
+AC_DEFUN([LESSTIF_HAVE_MAN2HTML],
 [
 AC_MSG_CHECKING(for suitable man2html)
 AC_PATH_PROG(man2html_cmd, man2html)
@@ -401,7 +401,7 @@
 dnl We check for the textmode web browsers 'lynx' and the less known 'links'.
 dnl Finally we try a simple sed script, a very poor fallback indeed.
 dnl
-AC_DEFUN([LT_HTML2TXT],
+AC_DEFUN([LESSTIF_HTML2TXT],
 [
 AC_PATH_PROG(LYNX_CMD,lynx)
 if test -x "$LYNX_CMD"
@@ -537,7 +537,7 @@
 CPPFLAGS="$CPPFLAGS $FREETYPE_CFLAGS"
 AC_CHECK_HEADERS(freetype/freetype.h,,, [#include <ft2build.h>])
 
-LT_HAVE_FREETYPE="no"
+LESSTIF_HAVE_FREETYPE="no"
 case "$ac_cv_header_freetype_freetype_h" in
 no)
 	CPPFLAGS="$saved_CPPFLAGS"
@@ -551,7 +551,7 @@
 		LIBS="$saved_LIBS"
 		;;
 	yes)
-		LT_HAVE_FREETYPE="yes"
+		LESSTIF_HAVE_FREETYPE="yes"
 		AC_DEFINE(HAVE_FREETYPE, 1, Means we have discovered the FreeType library)
 		AC_SUBST(FREETYPE_CFLAGS)
 		;;
@@ -591,7 +591,7 @@
 #
 # Check X configuration
 #
-LT_HAVE_XRENDER="no"
+LESSTIF_HAVE_XRENDER="no"
 case "$have_x" in
 yes)
 	XRENDER_CFLAGS="-I$x_includes"
@@ -616,7 +616,7 @@
 			LIBS="$saved_LIBS"
 			;;
 		yes)
-			LT_HAVE_XRENDER="yes"
+			LESSTIF_HAVE_XRENDER="yes"
 			AC_DEFINE(HAVE_XRENDER, 1, Means we have discovered the Xrender library)
 			AC_SUBST(XRENDER_CFLAGS)
 			AC_SUBST(XRENDER_LIBS)
@@ -710,7 +710,7 @@
 dnl
 dnl Should this be conditionally defined ?
 dnl
-if test "$LT_HAVE_XRENDER" = "yes" -a "$LT_HAVE_FREETYPE" = "yes"
+if test "$LESSTIF_HAVE_XRENDER" = "yes" -a "$LESSTIF_HAVE_FREETYPE" = "yes"
 then
 	AC_DEFINE(USE_XFT, 1, This means we will be using the Xft library)
 fi
--- origsrc/lesstif-0.95.0/configure.in	2006-06-10 04:22:48.000000000 -0500
+++ src/lesstif-0.95.0/configure.in	2008-10-26 22:18:19.350015900 -0500
@@ -255,8 +255,8 @@
 AM_PROG_LIBTOOL
 
 dnl Use an external malloc debugging library
-LT_WITH_DMALLOC
-LT_WITH_DBMALLOC
+LESSTIF_WITH_DMALLOC
+LESSTIF_WITH_DBMALLOC
 
 dnl
 dnl Some checks for required tools
@@ -346,7 +346,7 @@
 dnl Not only do we have to figure out whether we have a gethostname(),
 dnl some systems hide it in -lnsl so we have to figure out to link
 dnl it in when appropriate.
-LT_HAVE_GETHOSTNAME
+LESSTIF_HAVE_GETHOSTNAME
 
 dnl
 dnl Set the CFLAGS variable to the location of the found X-header files
@@ -379,7 +379,7 @@
 AM_CONDITIONAL(XIsR6, test x$Editres = xV6 )
 
 dnl Check for libXp (X11 Printing extensions)
-LT_HAVE_LIBXP
+LESSTIF_HAVE_LIBXP
 
 dnl
 dnl Whether to link -lXp
@@ -398,7 +398,7 @@
 dnl
 dnl Build libDtPrint (in lib/Dt) only if we have libXp and we're building a 2.1 version.
 dnl
-AM_CONDITIONAL(BuildLibDtPrint, test "$LT_HAVELIBXP" = "1" -a "$Build21" = "yes")
+AM_CONDITIONAL(BuildLibDtPrint, test "$LESSTIF_HAVELIBXP" = "1" -a "$Build21" = "yes")
 
 AC_PATH_MOTIF
 if test -n "$motif_includes"; then
@@ -444,7 +444,7 @@
 dnl check for man2html
 dnl
 dnl Hmm, a standard man2html may not be the right tool ...
-LT_HAVE_MAN2HTML
+LESSTIF_HAVE_MAN2HTML
 
 AC_MSG_CHECKING(whether to build the tests)
 BuildTest=""
--- origsrc/lesstif-0.95.0/test/configure.in	2006-06-10 04:22:48.000000000 -0500
+++ src/lesstif-0.95.0/test/configure.in	2008-10-26 22:18:05.412515900 -0500
@@ -165,13 +165,13 @@
 fi
 
 dnl Check for print extensions
-LT_HAVE_LIBXP
+LESSTIF_HAVE_LIBXP
 
 dnl Third party stuff
 AC_CHECK_HEADER(GL/GLwMDrawA.h,[AC_DEFINE(HAVE_GL_GLWMDRAWA_H)],[],[#include <X11/Intrinsic.h>])
 
 dnl malloc debugging
-LT_WITH_DMALLOC
+LESSTIF_WITH_DMALLOC
 
 AC_PATH_MOTIF
 if test -n "$motif_includes"; then
