--- origsrc/lesstif-0.95.0/acinclude.m4	2005-11-01 02:14:56.000000000 -0600
+++ src/lesstif-0.95.0/acinclude.m4	2006-06-15 17:49:39.031250000 -0500
@@ -463,7 +463,7 @@
 dnl AC_ARG_WITH(expat_includes,    [  --with-expat-includes=DIR     Use Expat includes in DIR], expat_includes=$withval, expat_includes=yes)
 dnl AC_ARG_WITH(expat_lib,         [  --with-expat-lib=DIR          Use Expat library in DIR], expat_lib=$withval, expat_lib=yes)
 AC_ARG_WITH(default_fonts,     [  --with-default-fonts=DIR      Use fonts from DIR when config is busted], defaultfonts="$withval", default_fonts=yes)
-dnl AC_ARG_WITH(confdir,           [  --with-confdir=DIR            Use DIR to store configuration files (default /etc/fonts)], confdir="$withval", confdir=yes)
+AC_ARG_WITH(confdir,           [  --with-confdir=DIR            Use DIR to store configuration files (default /etc/fonts)], confdir="$withval", confdir=yes)
 AC_ARG_WITH(fontconfig_includes, [  --with-fontconfig-includes=DIR  Use Fontconfig includes in DIR], fontconfig_includes=$withval, fontconfig_includes=yes)
 AC_ARG_WITH(fontconfig_lib,      [  --with-fontconfig-lib=DIR       Use Fontconfig library in DIR], fontconfig_lib=$withval, fontconfig_lib=yes)
 AC_ARG_WITH(fontconfig_config,   [  --with-fontconfig-config=PROG  Use Fontconfig configuration program PROG], fontconfig_config=$withval, fontconfig_config=yes)
@@ -473,7 +473,7 @@
 yes)
 	;;
 *)
-	X_FONT_DIR="$x_libraries/X11/fonts"
+	X_FONT_DIR="/usr/share/fonts"
 	AC_DEFINE_UNQUOTED(X_FONT_DIR,$X_FONT_DIR)
 	;;
 esac
@@ -594,8 +594,8 @@
 LT_HAVE_XRENDER="no"
 case "$have_x" in
 yes)
-	XRENDER_CFLAGS="-I$x_includes"
-	XRENDER_LIBS="-L$x_libraries -lXft -lXrender"
+	XRENDER_CFLAGS="$X_CFLAGS"
+	XRENDER_LIBS="$X_LIBS -lXft -lXrender"
 
 	saved_LIBS="$LIBS"
 	LIBS="$LIBS $XRENDER_LIBS"
@@ -722,3 +722,5 @@
 dnl End of special macros.
 dnl
 dnl
+sinclude([ac_debug.m4])
+sinclude([ac_have_libxp.m4])
--- origsrc/lesstif-0.95.0/lib/Xm-2.1/Print.c	2002-04-30 17:05:16.000000000 -0500
+++ src/lesstif-0.95.0/lib/Xm-2.1/Print.c	2006-06-14 18:47:41.953125000 -0500
@@ -77,11 +77,11 @@
    Also there's no working libXp. Many reasons to neglect this for now!
    Similar problems for cygwin.
  */
-#if defined(__EMX__) || defined(CYGWIN)
+#if defined(__EMX__)
 #ifdef UNIXLIKE
 #undef UNIXLIKE
 #endif
-#endif /* defined(__EMX__) || defined(CYGWIN) */
+#endif /* defined(__EMX__) */
 
 
 
--- origsrc/lesstif-0.95.0/test/acinclude.m4	1969-12-31 18:00:00.000000000 -0600
+++ src/lesstif-0.95.0/test/acinclude.m4	2006-06-14 18:47:42.031250000 -0500
@@ -0,0 +1 @@
+sinclude([../acinclude.m4])
